nixos/roles/github-actions-runner.nix

```
@@ -97,6 +97,15 @@ in
       default = [ ];
     };
 
+    extraLabels = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      description = lib.mdDoc ''
+        Extra labels to add to the runners to be able to target them.
+      '';
+      default = [ ];
```

Maybe make the default easy to override?

```suggestion
      default = ["nix"];
```